packages/@lwc/shared/src/html-attributes.ts

```diff
@@ -72,50 +73,39 @@ export function isGlobalHtmlAttribute(attrName: string): boolean {
     return GLOBAL_ATTRIBUTE.has(attrName);
 }
 
-const HTML_ATTRIBUTES_TO_PROPERTY: Record<string, string> = {
-    accesskey: 'accessKey',
-    readonly: 'readOnly',
-    tabindex: 'tabIndex',
-    bgcolor: 'bgColor',
-    colspan: 'colSpan',
-    rowspan: 'rowSpan',
-    contenteditable: 'contentEditable',
-    crossorigin: 'crossOrigin',
-    datetime: 'dateTime',
-    formaction: 'formAction',
-    ismap: 'isMap',
-    maxlength: 'maxLength',
-    minlength: 'minLength',
-    novalidate: 'noValidate',
-    usemap: 'useMap',
-    for: 'htmlFor',
+const HTML_PROPERTIES_TO_ATTRIBUTE: Record<string, string> = {
+    accessKey: 'accesskey',
+    readOnly: 'readonly',
+    tabIndex: 'tabindex',
+    bgColor: 'bgcolor',
+    colSpan: 'colspan',
+    rowSpan: 'rowspan',
+    contentEditable: 'contenteditable',
+    crossOrigin: 'crossorigin',
+    dateTime: 'datetime',
+    formAction: 'formaction',
+    isMap: 'ismap',
+    maxLength: 'maxlength',
+    minLength: 'minlength',
+    noValidate: 'novalidate',
+    useMap: 'usemap',
+    htmlFor: 'for',
 };
 
-const HTML_PROPERTIES_TO_ATTRIBUTE: Record<string, string> = {};
-keys(HTML_ATTRIBUTES_TO_PROPERTY).forEach((attrName) => {
-    const propName = HTML_ATTRIBUTES_TO_PROPERTY[attrName];
-    HTML_PROPERTIES_TO_ATTRIBUTE[propName] = attrName;
-});
+const CAPS_REGEX = /[A-Z]/g;
 
-export function htmlAttributeToProperty(attrName: string): string {
```

Comment:
This `htmlAttributeToProperty` is not used anywhere. I got rid of it, we might add it back in the future if needed.